packages/meditrak-server/src/routes/AccessRequests/assertAccessRequestPermissions.js

```diff
@@ -10,7 +10,7 @@ import {
 
 export const assertAccessRequestPermissions = async (accessPolicy, models, accessRequest) => {
   const results = await filterAccessRequestsByPermissions(accessPolicy, [accessRequest], models);
-  if (results.length === 0) {
+  if (results === 0) {
```

Comment:
Can you explain this one?

Reply:
Forgot to update my branch after applying that fix in the previous PR, fixed.